statsmodels/graphics/tests/test_functional.py

```diff
@@ -21,7 +22,7 @@
 data = data.raw_data[:, 1:]
 
 
-@skipif(not have_matplotlib, reason='matplotlib not available')
+@pytest.mark.skipif(not have_matplotlib, reason='matplotlib not available')
```

Comment:
just import the skipif name would be simpler
`from pytest.mark import skipif`
or whatever the correct import is

Reply:
I tried to be coherent in all marks, either, skip, skipif, or example.  Probably a few others to clean up, e.g., @slow

Reply:
I have made all marks consistent now using `@pytest.mark.`_marktype_